src/Plugin.php

```diff
@@ -267,7 +268,28 @@ private static function renderTargetMethods(array $methods): string
      */
     private static function renderArguments(array $array): string
     {
-        return '[' . implode(', ', array_map(fn($v) => var_export($v, true), $array)) . ']';
+        if (self::array_is_list($array)) {
```

Comment:
What's the reason to walk through whole array and use `var_export` on each element instead of calling `var_export` on the `$array` itself?
Even if the `$array` is integer indexed array `[0 => $v1, 1 => $v2]` still works when you initialise the result later.

Reply:
I wanted to have pretty code, but I gave up on it and I'm using only `var_export` in the last commit.